packages/core/stories/spinner/spinner.doc.stories.mdx

```diff
@@ -38,6 +38,31 @@ Set the `size` prop to `"large"` to use the large Spinner.
   <Story id="core-spinner--large" />
 </Canvas>
 
+## Usage
+
+### When to use Spinner
```

Comment:
I think "When to use Spinner" makes sense here since we're not talking about a general spinner but the Spinner component. If we wanted to add something before "Spinner" I think "When to use the Spinner component" would work. This would require updating all the other component pages.

Reply:
Yes, 'When to use (Component name)' is the standard